### PR TITLE
[fix/github-translation-action] Fix nested translation pull

### PR DESCRIPTION
**Pull request by hosy**
## Description
Fixed duplicated nested directories after tx pull and broken git commit 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] Added an issue with details about all relevant changes in the [**iOS documentation repository**](https://github.com/owncloud/docs-client-ios-app/issues).
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Added changelog files for the fixed issues in folder changelog/unreleased

### CODE REVIEW

File: .github/workflows/translate.yml [hunk 1]
```diff
@@ -60,7 +60,8 @@ jobs:
 
       - name: update-repo-before-commit
         run: |
-          git submodule sync
+          rm -rf ios-sdk
+          git submodule deinit ios-sdk
```
Comment (TheOneRing) [inline .github/workflows/translate.yml]: hmm can't we just skip the submodule stuff?

**Review by felix-schwarz — APPROVED** [review]
Assuming that this will only sync the `ios-app` locales, this looks good to me.

File: .github/workflows/translate.yml (hunk 1 above)
Comment (hosy) [reply .github/workflows/translate.yml]: it was already removed before your comment

File: .github/workflows/translate.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
         run:  bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/en.lproj && tx push -s --skip"
 
       - name: l10n-pull
-        run: bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "cd ownCloud/Resources/ && tx pull --force --skip --all"
+        run: bash .github/workflows/run-in-docker.sh owncloudci/transifex:latest "tx pull -a --skip --minimum-perc=75 -f"
```
Comment (TheOneRing) [inline .github/workflows/translate.yml:38]: please use the --long args rest looks fine thx

Comment (hosy) [reply .github/workflows/translate.yml]: fixed

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**